api/scripts/certification/rescore-certifications.js

```diff
@@ -0,0 +1,51 @@
+import 'dotenv/config';
+
+import * as url from 'node:url';
+
+import { disconnect, knex } from '../../db/knex-database-connection.js';
+import { CertificationRescoringByScriptJob } from '../../src/certification/session-management/infrastructure/jobs/CertificationRescoringByScriptJob.js';
+import { logger } from '../../src/shared/infrastructure/utils/logger.js';
+
+const modulePath = url.fileURLToPath(import.meta.url);
+const isLaunchedFromCommandLine = process.argv[1] === modulePath;
+
+async function main(certificationCourseIds) {
+  const jobs = await _scheduleRescoringJobs(certificationCourseIds);
+
+  const errors = jobs.filter((result) => result.status === 'rejected');
+  if (errors.length) {
+    errors.forEach((result) => logger.error(result.reason, 'Some jobs could not be published'));
+    return 1;
+  }
+
+  logger.info(`${jobs.length} jobs successfully published`);
+  return 0;
+}
```

Comment:
Comparé à d'autres scripts, celui-ci comporte peu de `logger.info` qui donneraient des infos sur le déroulé du script. ([voir par exemple ce script d'import de centre pilote](https://github.com/1024pix/pix/blob/d4d96ce9a0656166d5de1308c955ad137c560881/api/scripts/certification/next-gen/import-complementary-alone-feature-pilot-certification-centers-from-csv.js))

Cela peut s'expliquer par le fait que le script ne fait pas grand chose à part publier des jobs pg boss et que le monitoring est assez costaud derrière que ce soit via le statut du job en BDD ou bien encore via datadog.

Serait-ce pertinent tout de même de rajouter des logs comme on peut retrouver dans d'autres scripts, du type :
- logger.info('Starting scrip ...')
- logger.info('Starts publishing job ...')
- logger.info('Whatever you want ...')

Reply:
Oui on pourrait, j'ai bien envie d'améliorer aussi ce log en mettant le ratio de jobs crées sur nb de course passés en paramètre

```suggestion
async function main(certificationCourseIds) {
  const jobs = await _scheduleRescoringJobs(certificationCourseIds);

  const errors = jobs.filter((result) => result.status === 'rejected');
  if (errors.length) {
    errors.forEach((result) => logger.error(result.reason, 'Some jobs could not be published'));
    return 1;
  }

  logger.info(`${jobs.length} jobs / ${certificationCourseIds/length} successfully published`);
  return 0;
}

Reply:
Done dans la précèdente PR